util/new_keyboard.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+
+import argparse
+from distutils.dir_util import copy_tree
+import fileinput
+import os
+import re
+import shutil
+import subprocess
+import sys
+
+def print_error(message):
+    print("[\033[0;91mERROR\033[m] " + message, file=sys.stderr)
+
+def prompt(message, default=""):
+    """
+    Asks the user for input, showing the default value, if any, in brackets.
+    """
+    if default:
+        message += f" [{default}]"
+    value = input(message + ": ")
+    return value if value else default
```

Comment:
```suggestion
    return value or default
```
This is a common idiom and is equivalent to the original statement.